django/db/backends/base/features.py

```diff
@@ -443,6 +443,15 @@ def supports_explaining_query_execution(self):
         """Does this backend support explaining query execution?"""
         return self.connection.ops.explain_prefix is not None
 
+    @cached_property
+    def supports_inspectdb(self):
+        with self.connection.cursor() as cursor:
+            try:
+                self.connection.introspection.get_table_list(cursor)
+            except NotImplementedError:
+                return False
+            return True
```

Comment:
This matches what `inspectdb` does:
https://github.com/django/django/blob/68859a9b45f0b9d472c9277dc61e54afacc0da38/django/core/management/commands/inspectdb.py#L47-L54
I debated whether to use this implementation or to make it a boolean that the developer has to set to opt out. I could be convinced either way.